linetools/isgm/io.py

```diff
@@ -179,3 +181,100 @@ def write_joebvp_from_components(comp_list, specfile, outfile):
         s = comp.repr_joebvp(specfile, flags=flags, b_default=b_val)  # still, b values from abslines take precedence if they exist
         f.write(s)
     f.close()
+
+
+def read_igmg_to_components(igmguesses_json):
```

Comment:
IGMGuesses is a pyigm construct.
I think this method needs to go there.

And, I'll note that IGMSightline probably 
already does most of what you want here.